docs/UserGuide.md

```diff
@@ -284,7 +286,13 @@ Format: `focus INDEX`
 * To switch to another Candidate's information, user will just need to type the new command and the new index of the candidate.
 
 ## Scheduling interviews
+**:information_source: Note:** 
 
+* The duration of an interview is fixed at 30 minutes.
+* Interviews that have expired will automatically be deleted from the interview schedule upon a restart of the application. An interview is considered
+to be expired once its duration has fully passed. (e.g. If an interview is scheduled at 5PM, the interview is deemed to have expired from 5:30PM onwards).
```

Comment:
hi @lzan98 sorry just one final comment on this - i saw the latest issue you opened and tagged enen in, does this mean we need to change `5:30pm` to `5:31pm` here!

Reply:
np! it's the other way round - it was originally 5.31PM but I have changed it to 5.30PM!  Remember that interview scheduled at 5PM are deemed to have expired at 530PM (and hence allowed to schedule another interview at 530PM, etc.)